concepticondata/conceptlists/Say-2020-130.tsv

```diff
@@ -0,0 +1,131 @@
+Say-2020-130-87	87	govern	GOVERN	382	X (govern) Y	P. runs our town.
+Say-2020-130-88	88	miss	MISS (SOMEBODY)	2328	X (miss) Y	P. misses M.
+Say-2020-130-89	89	follow	FOLLOW	1085	X (follow) Y	P. follows M.
+Say-2020-130-90	90	dismount			X (get down) from Y	P. got down from the horse.
+Say-2020-130-91	91	listen	LISTEN	1608	X (listen) to Y	(What is P. doing?) P. is listening to radio.
+Say-2020-130-92	92	obey	OBEY	1635	X (obey) Y	(M. is a mischievous boy, but) P. always listens to / obeys his mother.
+Say-2020-130-93	93	hear	HEAR	1408	X (hear) Y	P. hears the music.
+Say-2020-130-94	94	mix	MIX	2017	X (get mixed) with Y	(Some honey was put in a jar of milk and stirred.) The honey got mixed with the milk.
+Say-2020-130-95	95	look	LOOK	1819	X (look) at Y	P. is looking at the clouds (trying to determine if it will rain).
+Say-2020-130-96	96	take off	TAKE OFF	3538	X (take off) Y	P. took off his shirt.
+Say-2020-130-97	97	dream (sleeping)	DREAM (SOMETHING)	1920	X (dream) about Y	(P. sleeps) P. dreams about M.
+Say-2020-130-98	98	agree	AGREE	3643	X (agree) with Y	(M. told P. that their son should learn Irish.) P. agreed with M.
+Say-2020-130-99	99	have a quarrel	QUARREL (VERB)	3302	X (have a quarrel) with Y	P. had a quarrel with M.
+Say-2020-130-100	100	cost	COST	242	X (cost) Y	This cup costs 1 euro.
+Say-2020-130-101	101	shoot at	SHOOT	1172	X (shoot) at Y	P. shot at the bird. (He missed.)
+Say-2020-130-102	102	pour	POUR	1487	X (pour) Y	P. poured the sugar (into the cup).
+Say-2020-130-103	103	lose			X (lose) Y	P. lost his keys.
+Say-2020-130-104	104	sink	SINK (DESCEND)	1088	X (sink) in Y	(This tree has very heavy wood. I threw a log in the river.) The log sank in the water.
+Say-2020-130-105	105	kill	KILL	1417	X (kill) Y	P. killed M.
+Say-2020-130-106	106	hit	HIT	1433	X (hit) Y	P. hit M.
+Say-2020-130-107	107	kiss	KISS	578	X (kiss) Y	P. kissed M.
+Say-2020-130-108	108	read	READ	724	X (read) Y	P. read an interesting book.
+Say-2020-130-109	109	move (bodypart)			X (move) Y	P. moved his finger.
+Say-2020-130-110	110	respect	RESPECT (SOMEBODY)	385	X (respect) Y	P. respects M.
+Say-2020-130-111	111	be squeamish			X (be squeamish) about Y	P. is squeamish about dirty dishes.
+Say-2020-130-112	112	be content			X (be content) wit Y X (be content) with	(P.’s son does well at school.) P. is OK/content/satisfied with his son.
+Say-2020-130-113	113	fall in love			X (fall in love) with Y	P. fell in love with M.
+Say-2020-130-114	114	trust	TRUST (SOMEBODY)	3902	X (trust) Y	(P. and M. have a happy family. M. never deceives P.) P. trusts M.
+Say-2020-130-115	115	sympathise	SYMPATHIZE	3903	X (sympathise) with Y	(P.’s mother is very ill.) P. sympathises with his mother.
+Say-2020-130-116	116	envy	ENVY	40	X (envy) Y	(M. has a car, and P. doesn't). P. envies M.
+Say-2020-130-117	117	be angry	BE ANGRY	2223	X (be angry) with Y	P. is angry with M.
+Say-2020-130-118	118	be surprised			X (be surprised) at Y	(M. made P. an expensive gift.) P. was surprised at this gift.
```

Comment:
SURPRISED

Reply:
I was thinking about mapping such instances as 'be surprised', 'be different, 'be shy', etc. to property concepts but the list specifically includes predicates only so I decided to leave them unmapped. Although the questionnaire is originally compiled in English, there is its Russian version on the website, and the Russian words for these concepts are clear-cut verbs that I wouldn't link to adjectival concept sets.

Reply:
I see your point, but I would argue that that's a language specific thing. Adjectives can in languages that have them usually also be used predicatively (and verbs can be used attributively, e.g. as relative clauses), and so what inflectional category they are expressed by doesn't truly affect the meaning. Based on how we treat concepts like BE ALIVE (namely containing both 'to live' and 'to be alive', I would assume that this policy could be extended to property-like meanings of any kind. But it is a deeper question that we should perhaps discuss separately.

Reply:
For SURPRISE and DIFFERENT we have at least one mapping with "be" so I'd go with @Schweikhard's suggestion for now. @Kristina-Pianykh you can open an issue so that we discuss this in more detail. There are other "be" cases this may apply to.

Reply:
Sounds reasonable, I've mapped the property-like glosses for now. Thank you for sharing your ideas on the issue!